netbox/utilities/testing/views.py

```diff
@@ -151,7 +151,7 @@ def test_create_object_without_permission(self):
             with disable_warnings('django.request'):
                 self.assertHttpStatus(response, 403)
 
-        @override_settings(EXEMPT_VIEW_PERMISSIONS=['*'])
+        @override_settings(EXEMPT_VIEW_PERMISSIONS=['*', 'users.user', 'users.group'])
```

Comment:
To avoid duplicating references to individual models, we can override `EXEMPT_EXCLUDE_MODELS` as well:
```suggestion
        @override_settings(EXEMPT_VIEW_PERMISSIONS=['*'], EXEMPT_EXCLUDE_MODELS=[])
```